workshops/templates/workshops/event_create_form.html

```diff
@@ -12,8 +15,15 @@
     {% endif %}
 {% endblock %}
 
-{% load crispy_forms_tags %}
-
 {% block content %}
 {% crispy form form_helper %}
 {% endblock %}
+
+{% block extrajs %}
+{{ form.media }}
+<script type="text/javascript">
+$(document).ready(function(){
+    $('#id_start,#id_end').datepicker();
```

Comment:
Please add `{format: "yyyy-mm-dd"}` here - otherwise this plugin refuses to work properly.